apps/datahub/src/app/home/news-page/key-figures/key-figures.component.html

```diff
@@ -4,7 +4,7 @@
       class="py-[37px] pl-[47px] rounded-lg border bg-white mb-5 card-shadow cursor-pointer"
       [figure]="recordsCount$ | async"
       [icon]="'folder_open'"
-      title="catalog.figures.datasets"
+      [title]="'catalog.figures.datasets'"
```

Comment:
There is no title on this. It the input that is called 'title'. So it should be [title] so angular understand that it is the input.

Reply:
I'm 99% sure you can specify string inputs without brackets and quotes by the way, and it makes it more readable. Not a big deal obviously!

Reply:
No it is not.

First of all, I don't see how it makes it more readable just by removing the brackets.

But worse, this is wrong !  `title` is not a known tag for a custom component like `gn-ui-figure` AND even if it was, this could be misleading since here we want to pass the `@Input() title` from the `gn-ui-figure` component because we do some computation on it to display it correctly! 

And by the way visit [datahub](https://www.geocat.ch/datahub/news) and just look, it is broken on main.Go on home page and hover the figures components, it shows "74 catalog.figures.organisations" instead of "74 organizations" AND if you hover on different part of the component you don't have the same title.

Bur don't worry, this PR fixes it.

Reply:
But maybe I'm wrong.

Reply:
https://v17.angular.io/guide/property-binding#binding-to-a-property

> The brackets, [], cause Angular to evaluate the right-hand side of the assignment as a dynamic expression.
> Without the brackets, Angular treats the right-hand side as a string literal and sets the property to that static value.

As for the missing translation, that would be a good thing yes :)